buildSrc/src/main/java/org/elasticsearch/gradle/plugin/PluginPropertiesExtension.java

```diff
@@ -105,6 +107,14 @@ public void setHasNativeController(boolean hasNativeController) {
         this.hasNativeController = hasNativeController;
     }
 
+    public boolean isBootstrapOnly() {
```

Comment:
This naming is a bit confusing, specifically the "only" part. Other plugins are not "bootstrap" plugins, so this plugin is not "only" a bootstrap plugin. Perhaps instead we could have a "type" attribute or something like that which defaults to "isolated", and then have this alternative value "bootstrap"?

Reply:
I've changed to a "type" as you suggested.